### ISSUE TITLE
encryption - decryption tool

### ISSUE THREAD

**Issue by shubham2704** [codeezzi/utility#70]
#  Create a  encryption - decryption tool in html,css and javascript. 
### users can choose different methods of encryption. Add atleast 3 methods of encryption

- Add navbar in your tool's html file. You can copy the navbar from index.html.
- Add screenshot of your tool in, PR message and also in the folder of your tool
- Before Making the PR Pull the latest changes that have been made to the repository

**Note: Please Don't Make Any Changes In `index.html` AND `contributors.html` **
**Note: Only make changes to the folder that you have created for your tool/utility **
--

## check the options that you have completed,  

I have;
- [ ] Read contibuting.md
- [ ] Followed all the guidelines
- [ ] Followed the directory Structure
- [ ] updated readme.md 
- [ ] add your name in the footer of your tool.html file (if you created length conversion tool then add your name to length.html file you created)
- [ ] provide below mentioned details if you want your name to be added in contributors.html
- [ ] mail, twitter, github profile link.

**Comment by georgioupanayiotis** [comment]
hello @shubham2704 I can take this task please assign me

**Comment by atom19-i** [comment]
hey,is there anything i can work on,here?

**Comment by shubham2704** [comment]
@atom19-i Yes you can work on this one or you can choose from [here](https://github.com/codeezzi/utility/issues)
